python/cudf/cudf/tests/test_index.py

```diff
@@ -2645,8 +2645,10 @@ def test_isin_multiindex(data, values, level, err):
         )
 
 
+rng = np.random.default_rng(seed=0)
+
 range_data = [
-    range(np.random.randint(0, 100)),
+    range(rng.integers(0, 100)),
     range(9, 12, 2),
     range(20, 30),
     range(100, 1000, 10),
```

Comment:
TODO: Can we open an issue to turn this usage into a fixture, so that we don't do this at test collection time?

Reply:
Done in this pr itself.